nyaggle/validation/split.py

```diff
@@ -385,8 +385,16 @@ def _make_test_folds(self, X, y=None, groups=None):
             y (ndarray):  labels(default = None)
             groups (None): (default = None)
         """
+        from sklearn.utils.multiclass import type_of_target
```

Comment:
Since nyaggle already has a  dependency to sklearn, we can safely move this statement to the top of the file.

Reply:
`from sklearn.utils.multiclass import type_of_target` and `from sklearn.utils.validation import check_array` move to the top of the file.